kubectl-minio/README.md

```diff
@@ -32,11 +32,11 @@ Options:
 
 #### MinIO Tenant Creation
 
-Command: `kubectl minio tenant create --name TENANT_NAME --secret SECRET_NAME --servers SERVERS --volumes TOTAL_VOLUMES --capacity TOTAL_RAW_CAPACITY [options]`
+Command: `kubectl minio tenant create-yaml --name TENANT_NAME --secret SECRET_NAME --servers SERVERS --volumes TOTAL_VOLUMES --capacity TOTAL_RAW_CAPACITY [options] > tenant.yaml`
 
 Creates a MinIO Tenant based on the passed values.
```

Comment:
This isn't strictly true now - we are creating a YAML file that `kubectl ` can ingest and apply.

Reply:
That is, until a user runs `kubectl apply -f`, we have created nothing beyond the spec (is that the right terminology?)